src/Rings/mpolyquo-localizations.jl

```diff
@@ -2555,7 +2555,6 @@ end
 @attr Union{Int, NegInf} function dim(R::MPolyQuoLocRing{<:Any, <:Any, <:MPolyRing, <:MPolyRingElem, <:Union{MPolyComplementOfPrimeIdeal, MPolyComplementOfKPointIdeal}})
   P = prime_ideal(inverted_set(R))
   I = saturated_ideal(modulus(R))
-  dim(I) === -inf && return -inf
```

Comment:
This catches the case where the sum below is `-Inf - (-Inf)`. Not sure this can ever happen, though.